tests/unit/test_models.py

```diff
@@ -316,6 +317,36 @@ def test_from_dict(self):
             str(err.exception), "Native transfers must have null token_address"
         )
 
+    def test_as_multisend_tx(self):
```

Comment:
This test is nice, but you don't implement as_multisend_tx in this PR, right? Maybe these tests should go into the other PR?

Reply:
Yes I was thinking this PR would be merged into its base. Just wanted to keep it separate since to make it easier to review. If you would prefer I can merge them now.